spec/features/schools/participants/add_participants/add_previously_withdrawn_mentor_spec.rb

```diff
@@ -296,8 +304,8 @@ def then_i_am_taken_to_mentor_start_training_page
     expect(page).to have_selector("h1", text: "When will George Mentor start their mentor training?")
   end
 
-  def when_i_choose_summer_term_2023
-    choose "Summer term 2023"
+  def when_i_choose_summer_term_next_cohort
+    choose "Summer term #{Cohort.next.start_year}"
```

Comment:
We only ask the question in the grey area outside of automatic assignment IIRC, the options would be Summer Term this cohort, Autumn next cohort or Spring next cohort, if that makes sense

Reply:
addressed in https://github.com/DFE-Digital/early-careers-framework/pull/4257/commits/868b5d580cc68deb3cb41ab519d55f2aa2d93d03 hopefully I got all of them